translations/management/commands/add_translations.py

```
@@ -0,0 +1,214 @@
+            if text and text.strip():
+                by_text.setdefault(text, []).append(translation_obj)
+
+        unique_texts = list(by_text.keys())
+        if not unique_texts:
+            self.stdout.write(self.style.SUCCESS("No non-empty English text to translate."))
+            return
+
+        try:
+            # "__all__" expands to every configured non-default language inside the client.
+            results = translate.bulk_translate(["__all__"], unique_texts)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)